src/main/java/org/folio/rest/impl/PostOrdersHelper.java

```diff
@@ -133,4 +141,11 @@ public Void handleError(Throwable throwable) {
 
     return null;
   }
+
```

Comment:
Although there is no hard requirement on the alpha text, do we want to have a consistent alpha-numeric naming convention? Just to keep this in mind because right now you just want to have something.

Reply:
I think we might need to do this later in order to handle the existing order case.  For now I think it's ok to punt on this just so we can move forward.